tests/fgnhg_test.py

```diff
@@ -9,187 +9,49 @@
-|                 |                    | 20             |
-|                 |                    | 21             |
-|                 |                    | 22             |
-|                 |                    | 23             |
-|                 |                    | 24             |
-|                 |                    | 25             |
-|                 |                    | 26             |
-|                 |                    | 27             |
-|                 |                    | 28             |
-|                 |                    | 29             |
-|                 |                    | 30             |
-|                 |                    | 31             |
-+-----------------+--------------------+----------------+
-| 100.50.25.12/32 | 200.200.200.4      | 0              |
-|                 |                    | 1              |
-|                 |                    | 2              |
-|                 |                    | 3              |
-|                 |                    | 4              |
-|                 |                    | 5              |
-|                 |                    | 6              |
-|                 |                    | 7              |
-|                 |                    | 8              |
-|                 |                    | 9              |
-|                 |                    | 10             |
-|                 |                    | 11             |
-|                 |                    | 12             |
-|                 |                    | 13             |
-|                 |                    | 14             |
-|                 |                    | 15             |
-+-----------------+--------------------+----------------+
-| 100.50.25.12/32 | 200.200.200.5      | 16             |
-|                 |                    | 17             |
-|                 |                    | 18             |
-|                 |                    | 19             |
-|                 |                    | 20             |
-|                 |                    | 21             |
-|                 |                    | 22             |
-|                 |                    | 23             |
-|                 |                    | 24             |
-|                 |                    | 25             |
-|                 |                    | 26             |
-|                 |                    | 27             |
-|                 |                    | 28             |
-|                 |                    | 29             |
-|                 |                    | 30             |
-|                 |                    | 31             |
-+-----------------+--------------------+----------------+
+FG NHG Prefix    Next Hop            Hash buckets
+---------------  ------------------  ------------------------------
+100.50.25.12/32  200.200.200.4       0   1   2   3   4   5   6   7
```

Comment:
Could we add one unit test where there are multiple prefixes utilizing the same next-hop IP to check if the output looks good in that scenario?

example:
  100.50.25.12/32 nh 200.200.200.5
   100.50.25.13/32 nh 200.200.200.5

Reply:
will add this UT in a follow up PR